spec/Section 3 -- Type System.md

```diff
@@ -477,14 +477,13 @@ value not representable by IEEE 754, a query error should be raised.
 
 ### String
 
-The String scalar type represents textual data, represented as UTF-8 character
-sequences. The String type is most often used by GraphQL to represent free-form
+The String scalar type represents textual data, represented as a sequence of Unicode code points. How the String is encoded internally (e.g. UTF-8) is left to the implementation. The String type is most often used by GraphQL to represent free-form
 human-readable text. All response formats must support string representations,
 and that representation must be used here.
 
 **Result Coercion**
 
-Fields returning the type {String} expect to encounter UTF-8 string internal values.
```

Comment:
I think this part should continue to specify UTF-8? As I read it, it's about the serialization of strings in responses, where specifying an encoding is actually appropriate?

Reply:
This section is about result coercion not serialization. Of course String can be serialized to UTF-8 (and often they are via UTF-8 JSON) but it doesn't have to be.